Assets/Mirror/Editor/Weaver/WeaverTypes.cs.meta

```diff
@@ -0,0 +1,3 @@
+﻿fileFormatVersion: 2
+guid: 2585961bf7fe4c10a9143f4087efdf6f
+timeCreated: 1596486854
```

Comment:
This meta file looks oddly small, other script files seem to have a MonoImporter block?

Reply:
because weaver isn't run in mono world